workflow/hosts/hera.yaml

```diff
@@ -10,10 +10,13 @@ noscrub: $HOMEDIR
 account: fv3-cpu
 scheduler: slurm
 queue: batch
-queue_service: service
+queue_service: batch
 partition_batch: hera
+partition_service: service
```

Comment:
Were these tested on Hera and Orion to confirm?

Reply:
I tested this on Hera.  I should also test on Orion.

Reply:
Confirmed that this also works on Orion.